Modules/_sqlite/module.c

```diff
@@ -417,8 +422,9 @@ PyMODINIT_FUNC PyInit__sqlite3(void)
        Now OptimizedUnicode is an alias for str, so it has no
        effect. */
     Py_INCREF((PyObject*)&PyUnicode_Type);
```

Comment:
This call looks like a ref leak.

Reply:
Please remove the (now) useless INCREF/DECREF dance.

Reply:
Got it